packages/core/types/@chainlink/index.d.ts

```diff
@@ -146,8 +146,18 @@ declare module '@chainlink/types' {
 
   export type RequiredInputParameter = boolean
   export type InputParameterAliases = string[]
+  export type InputParameter = {
+    aliases?: InputParameterAliases
+    description?: string
+    type?: 'bigint' | 'boolean' | 'array' | 'number' | 'object' | 'string'
+    required?: RequiredInputParameter
```

Comment:
Why are we making this a separate type instead of just calling it a `boolean`?

Reply:
I was originally trying to make use of the existing types, but tbh I think it's cleaner to just remove those. I'll add an update for it now